config/default/oembed_providers.provider.uiowa_kaltura.yml

```diff
@@ -0,0 +1,17 @@
+uuid: 30f705dd-7132-458e-82d6-ca3663e0fe9e
+langcode: en
+status: true
+dependencies: {  }
+id: uiowa_kaltura
+label: 'University of Iowa'
+provider_url: 'http://mediaspace.uiowa.edu'
+endpoints:
+  -
+    schemes:
+      - 'http://mediaspace.uiowa.edu/*'
```

Comment:
Is it necessary to allow http scheme here?

Reply:
not sure. i was just following some documentation way back and copied over from what we currently have.

Reply:
Ok. I just assumed that browsers would block mixed content from being embedded. I tried http and it gets embedded with a different https URL so I'm not sure this is an issue. Allowing http feeds odd.

Reply:
I guess since it gets rewritten as https, I'm ok with this.